examples/simple.rs

```diff
@@ -0,0 +1,74 @@
+use astarte_sdk::AstarteOptions;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Cli {
+    // Realm name
+    #[structopt(short, long)]
+    realm: String,
+    // Device id
+    #[structopt(short, long)]
+    device_id: String,
+    // Credentials secret
+    #[structopt(short, long)]
+    credentials_secret: String,
+    // Pairing URL
+    #[structopt(short, long)]
+    pairing_url: String,
+    // Interfaces directory
+    #[structopt(short, long)]
+    interfaces_directory: String,
+}
+
+#[tokio::main]
+async fn main() {
+    env_logger::init();
+
+    let Cli {
+        realm,
+        device_id,
+        credentials_secret,
+        pairing_url,
+        interfaces_directory,
+    } = Cli::from_args();
+
+    let mut sdk_options =
+        AstarteOptions::new(&realm, &device_id, &credentials_secret, &pairing_url);
+
+    sdk_options
+        .add_interface_files(&interfaces_directory)
+        .unwrap();
+
+    sdk_options.build().await.unwrap();
+
+    let mut device = sdk_options.connect().await.unwrap();
+
+    let w = device.clone();
```

Comment:
What happens to the MQTT client here? Will there be two separate clients with two connections after this call?

Reply:
![immagine](https://user-images.githubusercontent.com/87471904/132524020-a9a8482b-b666-4e7c-ac72-be19f578c2b9.png)
There should be only one client with one connection